tests/morphology.cpp

```diff
@@ -293,6 +297,47 @@ BOOST_AUTO_TEST_CASE(h5_read_v2)
 
     BOOST_CHECK(morphology.readPerimeters()->empty());
 }
+}
+
+BOOST_AUTO_TEST_CASE(h5_read_v2)
+{
+    boost::filesystem::path path(BBP_TESTDATA);
+    path /= "local/morphologies/14.07.10_repaired/v2/C010398B-P2.h5";
+
+    const brion::Morphology morphology(path.string());
+    _checkH5V2(morphology);
+}
+
+#ifdef BRION_USE_ZEROEQ
+BOOST_AUTO_TEST_CASE(zeroeq_read)
+{
+    zeroeq::Server server(zeroeq::NULL_SESSION);
+    server.handle(brion::ZEROEQ_GET_MORPHOLOGY, [](const void* data,
+                                                   const size_t size) {
+        if (!data || !size)
+            return zeroeq::ReplyData();
+
+        const std::string path((const char*)data, size);
+        const brion::Morphology morphology(path);
+        const brion::plugin::MorphologyZeroEQ serializable{morphology};
+        return zeroeq::ReplyData(serializable.getTypeIdentifier(),
```

Comment:
It doesn't match, but since it's ignored by the reply handler it does not matter. Fixed.